quicknes/bizinterface.cpp

```diff
@@ -224,118 +185,154 @@ EXPORT int qn_get_memory_area(Nes_Emu *e, int which, const void **data, int *siz
 		*data = e->high_mem();
 		*size = e->high_mem_size;
 		*writable = 1;
-		*name = "WRAM";
+		*name = "SRAM";

```

Comment:
Could we keep the old memory domain names?  Renaming them will break people's LUA scripts.

Reply:
Yes, I'm putting this back to WRAM